<launch>
    <node name="relocalization_sweeper_node" pkg="relocalization_sweeper" type="relocalization_sweeper_node" output="screen">
        <param name="voc_dir" type="string" value="$(find relocalization_sweeper)/voc/orbvoc.dbow3" />
        <param name="dataset_dir" type="string" value="$(find relocalization_sweeper)/rgb/" />
        <param name="pose_dir" type="string" value="$(find relocalization_sweeper)/pose/pose_list.txt" />
        <param name="depth_dir" type="string" value="$(find relocalization_sweeper)/depth/" />
        <param name="depth_threshold_max" type="double" value="6.0" />
        <param name="depth_threshold_min" type="double" value="0.6" />
        <param name="num_of_features" type="int" value="500" />
        <param name="scale_factor" type="double" value="1.2" />
        <param name="level_pyramid" type="int" value="5" />
    </node>
</launch>
